### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Lucas-Erkana**
- [x] Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- [x] Make sure that you used correct flow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
- [x] Following the list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
- [x] Add any missing JSX elements to your projects (without adding extra functionality)
- [x] Make sure the final design should match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/components/DisplayBookList.js [hunk 1]
```diff
@@ -14,8 +15,8 @@ const DisplayBookList = () => {
 
   return (
     <>
-      <div className="book_cont">
-        <ul>
+      <div className={styles.books_cont}>
+        <ul className={styles.books_min_cont}>
           {books.map((book) => (
             <DisplayAbook
               Id={book.Id}
```
Comment (fobadara) [inline src/components/DisplayBookList.js:22]: [REQUIRED] Kindly ensure that your design matches the template as much as possible.

image

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Lucash2022, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/DisplayBookList.js (hunk 1 above)
Comment (Lucas-Erkana) [reply src/components/DisplayBookList.js:22]: @fobadara 
Thank you so much for the review.
- [x] I have ensured that the colours match the design template at this [commit](https://github.com/Lucash2022/Bookstore/commit/727c5a25a6d3c3eae8bac424c2a9deda59055ea5)
- [x] I have included the image icon at this [commit](https://github.com/Lucash2022/Bookstore/commit/55f36374ea4e120f86b61285f5c4ce5ab4fbe677)

I

**Review by algerina — APPROVED** [review]
Hi @Lucash2022 ,

Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**